app/templates/public/speakers.hbs

```diff
@@ -4,7 +4,7 @@
       {{public/speaker-item speaker=speaker}}
     </div>
   {{/each}}
-  {{#infinity-loader infinityModel=model}}
+  {{#infinity-loader infinityModel=model triggerOffset=400 eventDebounce=50}}
```

Comment:
Let's remove the changes here then

Reply:
Sure!
Have we removed the codacy PR check?

Reply:
Yes